app/models/user.rb

```diff
@@ -27,6 +27,10 @@ def destroy_session!(session_identifier)
     client_sessions.where(id: session_identifier).destroy_all
   end
 
+  def joined?
+    !last_sign_in_at.nil?
```

Comment:
Can we do `last_sign_in_at.present?` I feel like it's easier to read.

Reply:
Obviously that's easier to read!